movieapp/src/components/movielist/error-boundary.tsx

```diff
@@ -0,0 +1,11 @@
+type ErrorProps = {
+  isOk: boolean;
+  children: JSX.Element;
+};
+
+function ErrorBoundary(props: ErrorProps): JSX.Element {
+  const ErrorText = (): JSX.Element => <h3>Error, please try again later</h3>;
+  return <>{props.isOk ? props.children : <ErrorText />}</>;
+}
+
+export default ErrorBoundary;
```

Comment:
good practice to use named export\import everywhere - it's more strict  and u can avoid different names issues

Reply:
export { ErrorBoundary }

Reply:
import { ErrorBoundary } from '../..'

Reply:
fixed